pkg/core/native/nativehashes/hashes.go

```diff
@@ -0,0 +1,34 @@
+package nativehashes
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Hashes of all native contracts.
+var (
+	Management  util.Uint160
```

Comment:
Just out of curiosity, can we have them precomputed automatically and saved as `const`s? `//go:generate` of some kind. We can combine them then with `nativenames` probably, consts don't cost a lot.

Reply:
Check out the last commit, it's an interesting experiment, but I still want to keep `Uint160` variables to reuse them in other code.

> We can combine them then with nativenames probably,

They have different names, is it OK? E.g. `nativenames.Gas` vs `nativehashes.GasToken`.